vic/drivers/cesm/bld/vic.buildnml.csh

```diff
@@ -0,0 +1,3 @@
+#! /bin/csh -f
+
+cd $RUNDIR
```

Comment:
You can probably just "cat" the file and then make sure it ends up in the $RUNDIR.  We might need to make some other changes so it also gets copied to the CaseDocs file.  That happens in preview_namelists.  You also should try to generate a vic.input_data.list file containing the filenames the model requires for this run.  That scripts check that all those files exist before trying to run as a way to exit gracefully on missing data.  You do not need to have ongoing restart file there.  In general, you should be able to get the coupling period from $LND_NCPL.  Have a look at the current vic.buildnml.csh for all these things, you should be able to replicate that example for your input file (vs a namelist file).  user_nl_vic is not useful.  We should make sure that file is not generated in the case directory.